kctf-operator/pkg/controller/challenge/challenge_default.go

```diff
@@ -0,0 +1,96 @@
+// File that set values not specified by the user to default
+package challenge
+
+import (
+	kctfv1alpha1 "github.com/google/kctf/pkg/apis/kctf/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	resource "k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	intstr "k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// Function to return the default values of autoscaling
+func AutoscalingDefault(chal *kctfv1alpha1.Challenge) kctfv1alpha1.AutoscalingSpec {
+	var AutoscalingDefault = kctfv1alpha1.AutoscalingSpec{
+		MinReplicas:                    1,
+		MaxReplicas:                    1,
+		TargetCPUUtilizationPercentage: 50,
+	}
+	return AutoscalingDefault
+}
+
+// Functions to return the default values
+func PersistentVolumeClaimsDefault(chal *kctfv1alpha1.Challenge) corev1.PersistentVolumeClaimList {
+	var persistentVolumeClaimsDefault = corev1.PersistentVolumeClaimList{
+		Items: []corev1.PersistentVolumeClaim{
+			corev1.PersistentVolumeClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: chal.Name,
+				},
+				Spec: corev1.PersistentVolumeClaimSpec{
+					Resources: corev1.ResourceRequirements{
+						Requests: corev1.ResourceList{
+							// Set 1Gi for the user
+							"storage": *resource.NewQuantity(1*1024*1024*1024*1024, resource.BinarySI),
```

Comment:
Isn't this 1Ti instead of 1Gi as the comment says?
Maybe you could use ParseQuantity here instead and we wouldn't need the comment anymore

Reply:
Yes, it's true, I put it wrong. I will have a look at ParseQuantity and fix this.